services/getUserIdsFromRoleId.js

```diff
@@ -0,0 +1,21 @@
+const firestore = require("../utils/firestore");
+const memberRoleModel = firestore.collection("member-group-roles");
+const usersModel = firestore.collection("users");
+
+export const getUserIdsFromRoleId = async (roleId) => {
```

Comment:
please add a try catch block in case something goes un expected

Reply:
![image](https://github.com/Real-Dev-Squad/website-backend/assets/61110378/6a529d7a-59ac-4d92-8471-4899468aee1f)